packages/preview/tud-letter/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+entrypoint = "src/lib.typ"
+name = "tud-letter"
```

Comment:
If you want this to be an official package, a faculty member needs to send an email to hello@typst.app endorsing this package. Otherwise the name needs to be prefixed by some non-descriptive part, so that multiple packages can co-exist. See our [naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules).